packages/editor/src/components/sidebar/index.js

```diff
@@ -176,20 +194,16 @@ const Sidebar = ( { extraPanels, onActionPerformed } ) => {
 	);
 
 	return (
-		<Tabs
+		<SidebarContent
+			tabName={ tabName }
```

Comment:
I think the tab-related data here should get moved within the SidebarContent rather than passed as props. I think it was done here previously because the Tabs needed be wrapping the SidebarContent. Now that we have a different structure, I think we could move it within.